index.html

```diff
@@ -68,7 +68,7 @@ <h2 class="primary-header">Glad to see you!</h2>
       </div>
     </section>
     <!-- cards section -->
-    <section  class="wrapper card">
+    <section class="wrapper card">
```

Comment:
- [ ] I think it is one of the main requirements is to create the original cards that you already created in HTML you should  create it by using javascript so kindly create it by that would help you learning how to create HTML by js 🔥🚀💯

![image](https://user-images.githubusercontent.com/58553711/188764823-a7ac4e1d-31a3-4a48-bf57-e03997c86971.png)